DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs

```diff
@@ -47,42 +47,42 @@ public sealed class ApplicationCommandsExtension : BaseExtension
         /// <summary>
         /// A list of methods for top level commands.
         /// </summary>
-        private static List<CommandMethod> _commandMethods { get; set; } = new List<CommandMethod>();
+        private static List<CommandMethod> CommandMethods { get; set; } = new List<CommandMethod>();
```

Comment:
privates should stick to _ I think, but we should discuss this.
I like it that way too

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#:~:text=When%20editing%20C%23%20code%20that%20follows%20these%20naming,t_.%20When%20writing%20method%20parameters%2C%20use%20camel%20casing.

Reply:
oh

Reply:
can we apply this completly with .editorconfig and run code cleanup, like I did with the other things before?

Reply:
honestly never fiddled with .editorconfig before sooooo mayyybeeee?

Reply:
Mhh let's look.

Reply:
let's see how many hundreds of files will get updated this time lol

Reply:
we do it on this branch tho.
Don't need 120k changes on the other one xD

Reply:
You don't want our browsers to break even more on the other one? lol